pandas/tests/arrays/masked/test_arrow_compat.py

```diff
@@ -64,3 +65,102 @@ def test_arrow_sliced():
     result = table.slice(2, None).to_pandas()
     expected = df.iloc[2:].reset_index(drop=True)
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.fixture
+def np_dtype_to_arrays(request):
+    import pyarrow as pa
```

Comment:
can we just add an import_optional_dependency at the top of this file, rather than having skips on every test.

Reply:
I used pyarrow 0.16.0 as the minimal version so `test_arrow_array` no longer gets run with pyarrow 0.15.0. Hope this is appropriate.